sympy/polys/series/flint_powerseriesring.py

```diff
@@ -0,0 +1,352 @@
+from __future__ import annotations
+
+from contextlib import contextmanager
+from typing import Any, TypeVar
+
+from sympy.polys.domains import Domain, QQ, ZZ
+from sympy.polys.series.powerseriesring import _series_from_list, PowerSeriesRing
+
+try:
+    from flint import ctx, fmpq_poly, fmpq_series, fmpz_poly, fmpz_series
+except ImportError:
+    pass
+
+ZZSeries = TypeVar("ZZSeries", bound='fmpz_series | fmpz_poly')
+QQSeries = TypeVar("QQSeries", bound='fmpq_series | fmpq_poly')
```

Comment:
I don't think it makes sense for this to be a TypeVar. It should just be:
```python
ZZSeries = fmpz_series | fmpz_poly
```
That can also be in a TYPE_CHECKING block if needed.